Add support for converting pyformat SQL to positional markers with bulk_parameters (#812)
crate/crate-python: docs/by-example/cursor.rst

    >>> [ desc[0] for desc in cursor.description ]
    ['name', 'position']

executemany() with named parameters
====================================

``executemany()`` also accepts a :class:`py:list` of :class:`py:dict` when

> the SQL uses ``%(name)s`` placeholders. The client converts both the SQL

Reviewer: ```suggestion
``executemany()`` also accepts a :class:`py:list` of :class:`py:dict` when
the SQL statement contains ``%(name)s`` placeholders. The client converts both the SQL
```

> template and all rows to positional format before sending to CrateDB:

Reviewer: ```suggestion
template and all rows to positional format before sending them to CrateDB:
```

>     >>> connection.client.set_next_response({

Author: ```suggestion
.. Hidden: set up mocked response

    >>> connection.client.set_next_response({
```

    ...     "results": [
    ...         {"rowcount": 1},
    ...         {"rowcount": 1}
    ...     ],
    ...     "duration": 123,
    ...     "cols": [],

>     ... })

Reviewer: Shouldn't this be hidden from the public docs given that it's mocking the response and isn't a public API that should/can be used?

Author: I added hidden keyword.
